examples/compare_product_prices/compare_product_prices.py

```diff
@@ -0,0 +1,68 @@
+"""This example demonstrates how to compare product prices across websites with query_data() method."""
+
+from agentql.ext.playwright.sync_api import Page
+from playwright.sync_api import sync_playwright
+
+# Set the URL to the desired website
+BESTBUY_URL = "https://www.bestbuy.com/site/nintendo-switch-oled-model-w-joy-con-white/6470923.p?skuId=6470923"
+TARGET_URL = "https://www.target.com/p/nintendo-switch-oled-model-with-white-joy-con/-/A-83887639#lnk=sametab"
+NINETENDO_URL = "https://www.nintendo.com/us/store/products/nintendo-switch-oled-model-white-set/"
+
+# Define the queries to get the product price
+PRODUCT_INFO_QUERY = """
+{
+    nintendo_switch
+    {
+        price
+    }
+}"""
+
+
+def print_header():
+    """Prints the header for the data table"""
+    print(f"{'Website':<25} | {'Product ':<20} | {'Price ':<20} ")
```

Comment:
Let's use `https://pypi.org/project/prettytable/` to not reinvent the wheel

Reply:
I'm a little hesitant to use `prettytable` library here because it's not included in Python standard library and requires `pip install prettytable`. I guess it depends on what kind of customer journey we want to provide to users. If we expect users to find this example through link on our documentation website and then copy / paste the code into their project. Then any extra setup requirement would create an issue.

Reply:
I think it's a matter of providing clear instructions. In reality people would not use our code to print tables, they probably won't print tables to console at all, tbh. You can rewrite this code in a shape of preparing the data first (which everyone can copy-paste, and then outputting them second (which can be ignored).

Reply:
Understood! You're absolutely right that people would not print tables. In this case, I will probably remove the outputting table part entirely. Because I feel like even with clear instructions, there will be a fairly large amount of users who will just copy/paste the code, run it, find out the code does not work, and then try to figure out what's going on. And the purpose of these examples is just to demonstrate the core functionality of AgentQL.